pkg/network/tracer/tracer_test.go

```diff
@@ -390,14 +390,21 @@ func (s *TracerSuite) TestTCPConnsReported() {
 	defer c.Close()
 	<-processedChan
 
-	// Test
-	connections := getConnections(t, tr)
-	// Server-side
-	_, ok := findConnection(c.RemoteAddr(), c.LocalAddr(), connections)
-	require.True(t, ok)
-	// Client-side
-	_, ok = findConnection(c.LocalAddr(), c.RemoteAddr(), connections)
-	require.True(t, ok)
+	if tr.config.EnableEbpfless {
+		time.Sleep(time.Millisecond * 500)
```

Comment:
Do we need this given the `require.Eventually` below?

Reply:
Ah whoops, I meant for the require.Eventually to replace this - thanks